### PR TITLE
Implement custom hooks for home page

### PR DESCRIPTION
**Pull request by mbantwerpes**

### CODE REVIEW

File: src/app/pages/Home/Home.tsx [hunk 1]
```diff
@@ -1,154 +1,44 @@
...
-    setLatestCards(newLatestCards);
+  function handleBookmarkClick(id: number) {
+    let newBookmarked = null;
+    if (bookmarked.includes(id)) {
+      newBookmarked = bookmarked.filter((element) => element !== id);
+      setBookmarked(newBookmarked);
+    } else {
+      newBookmarked = [...bookmarked, id];
+      setBookmarked(newBookmarked);
+    }
```
[the remark this answers is not in the document]
Comment (dirkzukunft) [reply src/app/pages/Home/Home.tsx]: or a const for the parameter in line 23? ;)

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Cheers

File: src/app/pages/Home/Home.tsx [hunk 2]
```diff
@@ -1,154 +1,41 @@
...
   }
 
   return (
     <div className={styles.container}>
       <div className={styles.innerContainer}>
         <Header className={styles.header}>Top Five</Header>
-
         <section className={styles.topFiveCards}>
-          {topFiveCards.map((card) => {
+          {latestMovies?.map((element) => {
```
Comment (marwinburesch) [inline src/app/pages/Home/Home.tsx:30]:
```suggestion
          {latestMovies?.map((movie) => {
```

File: src/app/pages/Home/Home.tsx [hunk 3]
```diff
@@ -162,12 +49,15 @@ export default function Home(): JSX.Element {
             </div>
           </div>
           <div className={styles.latestContainer}>
-            {latestCards.map((card) => {
+            {topFiveMovies?.map((element) => {
```
Comment (marwinburesch) [inline src/app/pages/Home/Home.tsx:52]:
```suggestion
            {topFiveMovies?.map((movie) => {
```

**Review by marwinburesch — CHANGES_REQUESTED** [review]

**Review by marwinburesch — APPROVED** [review]
💯

**Pull request merged**